submanifests/optional.yaml

```diff
@@ -40,7 +40,7 @@ manifest:
       groups:
         - optional
     - name: tf-m-tests
-      revision: d552e4f18b92032bd335d5e3aa312f6acd82a83b
+      revision: pull/11/head
```

Comment:
State that it's to 2.1.1 in the commit message. Also, can you add a line about that to the release notes? I can do that first in my PR to main and you can copy from there.

Reply:
Will do it properly if CI passes, just want to make sure the two are compatible before moving these changes over to the MCUboot repo